script.js

```diff
@@ -164,3 +165,15 @@ for (let index = 0; index < openPopup.length; index += 1) {
 closePopup.addEventListener('click', () => {
   modalPopup.style.display = 'none';
 });
+
+/* Validate email form */
+const email = document.getElementById('email').value;
+const lowerCase = email.toLowerCase;
+const submit = document.querySelector('.submit');
+
+submit.addEventListener('click', (e) => {
```

Comment:
- [ ] Even if the email is in lowercase the error message still shows and the form doesn't get submitted, kindly fix this so the message show only when the email is not in lowercase
- [ ] As the previous reviewer mentioned, kindly consider moving the error message close to the submit button

![Screenshot_20220909_181311](https://user-images.githubusercontent.com/25227022/189407949-ceb2beb3-b43d-461a-81f5-50f5d344d7a7.png)